manifests/k/KDE/Dolphin/21.04.3/KDE.Dolphin.installer.yaml

```
@@ -1,16 +1,16 @@
 # Created using wingetcreate 0.4.1.1
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 
-PackageIdentifier: KDE.Dolphin
-PackageVersion: 21.04.3
+PackageIdentifier: KDE.Dolphin.Release

```

```suggestion
PackageIdentifier: KDE.Dolphin
```
This isn't needed here.